devops/makeLambdaFunction/makeLambdaFunction.yml

```diff
@@ -0,0 +1,26 @@
+Parameters: 
+  LambdaName: 
+    Description: Unique name of your Lambda function.
+    Type: String
+    
+Resources:
+  makeLambdaFunction:
+    Type: AWS::Lambda::Function
+    Properties:
+      FunctionName: !Ref LambdaName
+      Role: arn:aws:iam::061431082068:role/Lambda_Role
+      Runtime: nodejs14.x
+      Handler: index.handler
+      Code:
+        ZipFile: |
```

Comment:
I was curious about how we might be able to pass a file here and found this SO post that might be useful in the future: 

[https://stackoverflow.com/questions/54876833/upload-local-file-on-parameter-zipfile-awslambdafunction](https://stackoverflow.com/questions/54876833/upload-local-file-on-parameter-zipfile-awslambdafunction)

Reply:
Nice, thanks for the resource @dansjack !